MAUI/TabView/Getting-Started.md

```diff
@@ -210,4 +210,250 @@ namespace TabViewMauiSample
 
 ![Getting_Started](images/Getting_Started.png)
 
-N> View [sample](https://github.com/SyncfusionExamples/maui-tabview-samples/tree/main/TabViewGettingStarted) in GitHub
\ No newline at end of file
+N> View [sample](https://github.com/SyncfusionExamples/maui-tabview-samples/tree/main/TabViewGettingStarted) in GitHub
+
+## Populate ItemsSource
+
+Items can be added to the control using the `ItemsSource` property of [SfTabView](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.TabView.SfTabView.html).
+
+Using ItemsSource, objects of any class can be given as items for SfTabView. The views corresponding to the objects can be set using the HeaderItemTemplate for the header items and ContentItemTemplate for the content. 
+
+Create a model class with TabItems collection property initialized with required number of data objects   as shown in the following code example.
+
+{% tabs %}
+
+{% highlight C# %}
+
+    public class Model: INotifyPropertyChanged
+    {
+
+    public event PropertyChangedEventHandler PropertyChanged;
+
+    protected void OnPropertyChanged(string propertyName)
+    {
+        var handler = PropertyChanged;
+        if (handler != null)
+            handler(this, new PropertyChangedEventArgs(propertyName));
+    }
+
+    private string name;
+
+    public string Name
+    {
+        get { return name; }
+        set
+        {
+            name = value;
+            OnPropertyChanged("Name");
+        }
+    }
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+{% tabs %}
+
+{% highlight C# %}
+
+ public class TabItemsSourceViewModel:INotifyPropertyChanged
+    {
+    public event PropertyChangedEventHandler PropertyChanged;
+
+    protected void OnPropertyChanged(string propertyName)
+    {
+        var handler = PropertyChanged;
+        if (handler != null)
+            handler(this, new PropertyChangedEventArgs(propertyName));
+    }
+
+    private ObservableCollection<Model> tabItems;
+    public ObservableCollection<Model> TabItems
+    {
+        get { return tabItems; }
+        set
+        {
+            tabItems = value;
+            OnPropertyChanged("TabItems");
+        }
+    }
+        public TabItemsSourceViewModel()
+        {
+            TabItems = new ObservableCollection<Model>();
+            TabItems.Add(new Model() { Name = "Alexandar" });
+            TabItems.Add(new Model() { Name = "Gabriella" });
+            TabItems.Add(new Model() { Name = "Clara"});
+            TabItems.Add(new Model() { Name = "Tye" });
+            TabItems.Add(new Model() { Name = "Nora" });
+            TabItems.Add(new Model() { Name = "Sebastian" });
+           
+        }
+
+  }
+
+{% endhighlight %}
+
+{% endtabs %}
+
+The following code example binds the collection to the SfTabView.ItemsSource property.
```

Comment:
The following code example binds the collection to the `SfTabView.ItemsSource` property.